### DISCUSSION TITLE
Works fine in RN?

### DISCUSSION THREAD

**Discussion by devhyunjae** [TanStack/query#2008] [Q&A] [answered]
https://react-query.tanstack.com/react-native

```
React Query is designed to work out of the box with React Native,
with an exception to the devtools, which are only supported with React DOM at this time.
If you would like to help us make the devtools platform agnostic, please let us know!
```

Does this mean, it works fine in RN? Except it doesn't support devtools? or is there other problems for production world?

**Comment by Andrey-Bazhanov** [comment, marked as answer]
I don't use RN personally, but library should work fine. You just need do one thing: https://react-query.tanstack.com/guides/window-focus-refetching#managing-focus-in-react-native

**Reply by nandorojo** [reply]
Also add this if you want connectivity refetches: https://react-query.tanstack.com/reference/onlineManager

**Comment by nandorojo** [comment]
It would be great to have devtools in react native here too. Is there a way to get that to work?
